### ISSUE TITLE
Yin-Yang doesn't start

### ISSUE THREAD

**Issue by Saraistupid** [oskarsh/Yin-Yang#200]
### Describe the bug

Yin-Yang doesn't start at all giving this error:
```
$ yin-yang 
Using default configuration values.
Traceback (most recent call last):
  File "/opt/yin-yang/main.py", line 13, in <module>
    from src import daemon_handler
  File "/opt/yin-yang/src/daemon_handler.py", line 7, in <module>
    from src.config import ConfigWatcher, config
  File "/opt/yin-yang/src/config.py", line 474, in <module>
    config = ConfigManager()
  File "/opt/yin-yang/src/config.py", line 179, in __init__
    self.load()
  File "/opt/yin-yang/src/config.py", line 246, in load
    pl.theme_light = config_loaded['plugins'][pl.name.lower()]['light_theme']
  File "/opt/yin-yang/src/plugins/konsole.py", line 42, in theme_light
    self.update_profile(False, value)
  File "/opt/yin-yang/src/plugins/konsole.py", line 174, in update_profile
    self.create_profiles()
  File "/opt/yin-yang/src/plugins/konsole.py", line 189, in create_profiles
    copyfile(self.user_path / self.default_profile, light_profile)
  File "/usr/lib/python3.10/shutil.py", line 254, in copyfile
    with open(src, 'rb') as fsrc:
FileNotFoundError: [Errno 2] No such file or directory: '/home/saraistupid/.local/share/konsole/Light.profile'
```

### Affected versions

- Yin-Yang version: 3.2.4-1 (from AUR)
- Python version: 3.10.9
- Qt version: 5.15.8

**Comment by l0drex** [comment]
Create a custom profile in konsole and then try again, maybe that helps.

**Comment by alexandervnuchkov** [comment]
To add to the report: 

Yin-Yang also crashes if 
- the konsole profile is present but does not contain either the 'Appearance' or the 'ColorScheme' entries
- the konsole profile is present but has spaces in the file name (e.g., 'Profile 1.profile')

**Comment by bphd** [comment]
> Create a custom profile in konsole and then try again, maybe that helps.

That is a workaround, the software needs to create and populate the needed files. For now the software is literally unable to run without tinkering. That should be considered critical

**Comment by l0drex** [comment]
Can you check if this is now fixed on the beta branch?

**Comment by bphd** [comment]
Still not resolved in 3.2.4.r9.g42ce4da-1
